test/utils.mjs

```diff
@@ -26,5 +27,22 @@ export async function createAosLoader(params) {
 }
 
 export async function createAntAosLoader() {
-  return createAosLoader(ANT_EVAL_OPTIONS);
```

Comment:
Is this function still used anywhere?

Reply:
Nope - removed along with some other constants.

Reply:
[091b4fc](https://github.com/ar-io/ar-io-ant-process/pull/54/commits/091b4fcb8de8f2f1a342d2578d225f6af707f2c6)